README.md

```diff
@@ -8,12 +8,38 @@ Note: `Kokkos` must be configured with `Kokkos_ENABLE_LIBDL=ON` to load profilin
 
 ## General Usage
 
-To use one of the tools you have to compile it, which will generate a dynamic library. Before executing the Kokkos application you then have to set the environment variable `KOKKOS_TOOLS_LIBS` to point to the dynamic library e.g. in the `bash` shell:
-```
-export KOKKOS_TOOLS_LIBS=${HOME}/kokkos-tools/src/tools/memory-events/kp_memory_event.so
-```
+To use one of the tools you have to compile it, which will generate a dynamic library. Before executing the Kokkos application you then have to set the environment variable `KOKKOS_TOOLS_LIBS` to point to the dynamic library. Many of the tools will produce an output file that uses the hostname as well as the process id as part of the filename.
+
+Use the route of either CMake or Makefile to build and run Kokkos Tools. The following provides instructions on both routes. 
+
+## Using cmake 
+
+### Build
+
+1. Create a build directory in Kokkos Tools, e.g., type `mkdir myBuild; cd myBuild`
+2. To configure, type `ccmake .. -DCMAKE_INSTALL_PREFIX=${YOUR_KOKKOS_TOOLS_INSTALL_DIR}` for any options you would like to enable/disable. 
+3. To compile, type `make`
+4. To install, type `make install`
+
+### Run 
+
+Given your installed tool shared library `lib<name_of_tool_shared_lib>.so` and an application executable called yourApplication.exe, type: 
+
+`export KOKKOS_TOOLS_LIBS=${YOUR_KOKKOS_TOOLS_INSTALL_DIR}/lib<name_of_tool_shared_lib>.so; ./yourApplication.exe`
```

Comment:
There is no reason to distinguish between Makefiles and CMake for running.
```suggestion
```

Reply:
There is a subtle difference. Maybe that should be changed in the build, but that's another PR.

The difference is that Makefiles generate `<name_of_tool_shared_lib>.so` in the source directory of the tool connector, while the cmake generates a library `lib<name_of_tool_shared_lib>.so` in the install directory.

Reply:
Homogenizing the library names sounds like worth fixing.

Reply:
> Homogenizing the library names sounds like worth fixing.

I agree with you and thanks for that feedback. 

This would be a separate PR and is outside the scope of this PR (if you disagree, let me know).

Note the updated and more specific goal of this PR that I put in the description, so it is clear what needs to be fixed in this PR.